src/display/Drawer/Drawer.spec.tsx

```diff
@@ -45,4 +51,95 @@ describe("Drawer", () => {
 
     expect(content).toBeInTheDocument();
   });
+
+  it("Should show the sides sheet", async () => {
+    const onCloseMock = vi.fn();
+
+    render(
+      <Drawer
+        isOpen={true}
+        onClose={onCloseMock}
+        header={<div>Test</div>}
+        detent="content"
+        isDismissable
+      >
+        <div>Content</div>
+      </Drawer>,
+    );
+
+    await waitFor(() => screen.getByText(/content/i));
+
+    const modal = screen.getByTestId("side-sheet");
+
+    expect(modal).toBeInTheDocument();
+  });
+
+  it("Should show the bottom sheet", () => {
```

Comment:
In this test, you check if the side sheet is not in the document, but you don't test if the bottom sheet is present no ?